scripts/apps/archive/index.js

```diff
@@ -135,8 +135,8 @@ angular.module('superdesk.apps.archive', [
                     return authoring.itemActions(item).unspike;
                 }]
             })
-            .activity('duplicate', {
-                label: gettext('Duplicate'),
+            .activity('duplicateHere', {
+                label: gettext('Duplicate Here'),
```

Comment:
`Duplicate Here` makes sense if the user is in the monitoring view of the desk. It would be confusing when you are in the custom workspace or global search. The duplicate item could land on your default desk as well.